docs/developers/how-to-build-plugins.md

```diff
@@ -2,82 +2,111 @@
 
 #### Params
 
-| Param    | Type            | Purpose                                                                        |
-| -------- | --------------- | ------------------------------------------------------------------------------ |
-| `inputs` | `ModelParams[]` | Array of data provided in the `inputs` field of a component in a manifest file |
+| Param    | Type             | Purpose                                                                        |
+| -------- | ---------------- | ------------------------------------------------------------------------------ |
+| `inputs` | `PluginParams[]` | Array of data provided in the `inputs` field of a component in a manifest file |
 
 #### Returns
 
-| Return value | Type                     | Purpose                                                    |
-| ------------ | ------------------------ | ---------------------------------------------------------- |
-| `outputs`    | `Promise<ModelParams[]>` | `Promise` resolving to an array of updated `ModelParams[]` |
+| Return value | Type                      | Purpose                                                     |
+| ------------ | ------------------------- | ----------------------------------------------------------- |
+| `outputs`    | `Promise<PluginParams[]>` | `Promise` resolving to an array of updated `PluginParams[]` |
 
-## What are `ModelParams`?
 
-`ModelParams` are a fundamental data type in the Impact Framework. The type is defined as follows:
+### What are `PluginParams`?
+
+`PluginParams` are a fundamental data type in the Impact Framework. The type is defined as follows:
 
 ```ts
-export type ModelParams = {
+export type PluginParams = {
   [key: string]: any;
 };
 ```
 
-The `ModelParams` type therefore defines an array of key-value pairs.
+The `PluginParams` type therefore defines an array of key-value pairs.
 
-IF needs to know about all the parameters used in each model pipeline. The default behaviour is that it grabs parameters from a local file, `params.ts`. This file defines the standard set of parameter names, their units, a descriptiona nd the method used to aggregate them across time or across a graph.
+IF needs to know about all the parameters used in each pipeline. The default behaviour is that it grabs parameters from a local file, `params.ts`. This file defines the standard set of parameter names, their units, a descriptiona nd the method used to aggregate them across time or across a tree.
 
-If your new plugin uses new parameters that are not included in `params.ts`, you can simply add them to your manifest file in a section named `params`. For example:
+If your new plugin uses new parameters that are not included in `params.ts`, you can smanifesty add them to your manifest file in a section named `params`. For example:
```

Comment:
@jmcook1186 smanifesty instead of simply

Reply:
```suggestion
If your new plugin uses new parameters that are not included in `params.ts`, you can simply add them to your manifest file in a section named `params`. For example:
```